src/main/java/Blu/command/UnmarkCommand.java

```diff
@@ -0,0 +1,45 @@
+package blu.command;
+
+import blu.exception.BluException;
+import blu.storage.Storage;
+import blu.task.Task;
+import blu.task.TaskList;
+import blu.ui.UI;
+
+/**
+ * Represents a command to unmark a task in the task list with a specified index.
+ */
+public class UnmarkCommand extends Command {
+    private final int taskIdx;
+
+    /**
+     * Constructs an UnmarkCommand to unmark the task at the specified index.
+     *
+     * @param taskIdx The index of the task to be unmarked.
+     */
+    public UnmarkCommand(int taskIdx) {
```

Comment:
I like how UnmarkCommand conveys its purpose, a command related to unmarking a task.